solr/core/src/java/org/apache/solr/storage/TeeDirectory.java

```diff
@@ -450,6 +457,15 @@ public long getFilePointer() {
     public long getChecksum() throws IOException {
       return primary.getChecksum();
     }
+
+    @Override
+    public long getBytesWritten() {
+      if (secondary instanceof CompressingDirectory.SizeReportingIndexOutput) {
+        return ((CompressingDirectory.SizeReportingIndexOutput) secondary).getBytesWritten();
+      } else {
+        return getFilePointer();
+      }
```

Comment:
I _think_ this instanceof fallback is ok, as far as semantics of `getBytesWritten()`? wdyt?